### PR TITLE
Update for final draft

### PR DESCRIPTION
**Pull request by karjac14**

### CODE REVIEW

File: journal-week-2.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 
 ###Proof of Life
 
-We made it through the first week, and we are still alive. the second week was like a recovery period for us after the shock.  Most of us started to understand how this course will look like. That we will be bombarded with assignment especially on weekends, and from there we will learn to solve problems though research, team work and alot of educated guesses. 
+We made it through the first week, and we are still alive. the second week was like a recovery period for us after the shock. Most of us started to understand how this course will look like - that we will be bombarded with assignment especially on weekends, and from there we will learn to solve problems through research, team work and alot of educated guesses.
```
Comment (Ricecoder) [inline journal-week-2.md]: You might want to capitalize the T in "The second week..." otherwise it's getting better.

**Comment by Ricecoder** [comment]
I really like how you talk about the adjustment from the first week to the second. Also it's great that you incorporated the term vanilla javascript in your essay. I had forgotten about that! :+1:

File: journal-week-2.md (hunk 1 above)
Comment (jeremyrist) [inline journal-week-2.md]: ...bombarded with ASSIGNMENTS especially on....

Comment (jeremyrist) [inline journal-week-2.md]: alot is 2 words: a lot...

File: journal-week-2.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
-Last week, we learned a lot:
+Last week, we learned a lot of new things:
 
-**More of Githubbing** - more exposure in terminal, solving conflicts. Using source tree as alternative.
+More of Githubbing - more exposure in CLI, solving conflicts. Using source tree as alternative. I thought I was comfortable using this VCS, but it seems everyday we encounter different conflicts and because of that we continue learning more about the system as we work with it.
 
-**Understanding JS** - continued improving our skills in JS by applying it to more practical ways. i.e. linking it with html.
+Understanding JS - continued improving our skills in JS by applying it to more practical ways. i.e. linking it with html and in a a way with css.
```
Comment (jeremyrist) [inline journal-week-2.md]: you have "a a" but it should be just one "a"

File: journal-week-2.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 
-**More testing of JS via node and browser** - accessing html, css and JS files in browser
+More testing of JS via node and browser - accessing html, css and JS files in browser particularly chrome dev tools.
 
-**DOM** - we had a deep introduction of the DOM, and had a great undertsnding of it now. We managed to control the DOM behaviours using vanilla javascript. I have learned a lot in this particular tasks by trying my best to understand pseudocoding and impeement it with javascript. I also learned here to work with other students and work as a team. 
+DOM - we had a deep introduction of the DOM, and had a great undertsnding of it now. We managed to control the DOM behaviours using vanilla javascript. I have learned a lot in this particular tasks by trying my best to understand pseudocoding and impeement it with javascript. I also learned last week to work with other students and work as a team.
```
Comment (jeremyrist) [inline journal-week-2.md]: we had a deep introduction of the DOM, and HAVE a great UNDERSTANDING of it now....

Behaviors is spelled without a "u"....

....particular TASK...not "tasks"...

..."impeement" should be "IMPLEMENT"

**Comment by jeremyrist** [comment]
:+1: You da man, always fun sitting next to you in class and working on the homework together! Keep up the good work!

**Comment by cmauban** [comment]
Yes, I'm glad we're all still alive! Great overview of what we've learned thus far. I can tell you definitely have a good understand of JavaScript! :+1:

**Pull request merged**